docs/usage.md

```diff
@@ -128,133 +146,119 @@ yarn run gulp test:acceptance # Run only acceptance (in-browser) tests on produc
+If the Python package requirements files have changed, 
+you will need to stop `docker-compose` (if it is running) 
+and rebuild the Python containers using:
 
-#### Handle updates to Python requirements
+- Python 2.7: `docker-compose build python2`
+- Python 3.6: `docker-compose build python3`
 
-If Compose is running, stop it with CTRL-C. Run:
+The next time you run `docker-compose up` the new requirements will be in place.
 
-`docker-compose build python`
+### Work on satellite apps
 
-This will update your Python image. The next time you run `docker-compose up`,
-the new requirements will be in place.
+If you need to work on any of our 
+[satellite apps](https://github.com/cfpb/cfgov-refresh/blob/master/requirements/optional-public.txt), 
+you will need to perform a `git clone` in the 
+[`develop-apps` directory](https://github.com/cfpb/cfgov-refresh/tree/master/develop-apps).
+These will automatically be added to the
+[PYTHONPATH](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH) 
+in the Python containers, 
+and apps contained within will be importable from Python running in the containers. 
 
-#### Set environment variables
+For any satellite apps that provide front-end assets that need to be built, 
+you will need to run that step seperately.
 
-Environment variables from your `.env` file are sourced when the python container
-starts and when you access a running container with `./shell.sh` Your shell
-environment variables, however, are not visible to applications running in Docker.
-To add new environment variables, simply add them to the `.env` file, stop compose
-with ctrl-c, and start it again with `docker-compose up`.
+For example, to work on [college-costs](https://github.com/cfpb/college-costs):
 
-#### Get familiar with Docker Compose, and our configuration
-
-docker-compose.yml contains a sort of "recipe" for running the site. Each entry
-in the Compose file describes a component of our application stack (Postgres,
-Elasticsearch, and Python), and either points to a public image on Dockerhub,
-or to a Dockerfile in cfgov-refresh. You can learn a lot more about Compose
-files in [the docs](https://docs.docker.com/compose/compose-file/)
-
-Similarly, a Dockerfile contains instructions for transforming some base image,
-to one that suits our needs. The Dockerfile sitting in the top level of
-cfgov-refresh is probably the most interesting. It starts with
-[the public CentOS:7 image](https://hub.docker.com/_/centos/), and installs
-everything else neccessary to run our Python dependencies and the Django app
-itself.  This file will only be executed:
+```bash
+# Check out into develop-apps:
+cd develop-apps
+git clone https://github.com/cfpb/college-costs
 
-- the first time you run `docker-compose up` (or the first time after you
-re-create the Docker Machine VM)
-- any time you run `docker-compose build`
+# Build the front-end:
+cd college-costs
+./setup.sh
+```
```

Comment:
What about installing their requirements?

Reply:
Good point — that has to happen in the container shell. I'll add that.

Reply:
Added in 0ea058b.